infrastructure/monitoring/prometheus/prometheus.yml

```
@@ -0,0 +1,150 @@
+# Prometheus Configuration for Luppa PLC Inventory System
+# Configured for production monitoring with comprehensive service discovery
+
+global:
+  # How frequently to scrape targets by default
+  scrape_interval: 15s
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)